examples/debounced.rs

```diff
@@ -28,7 +28,7 @@ fn main() {
 
     // print all events, non returning
     for events in rx {
-        for e in events {
+        if let Ok(e) = events {
             println!("{:?}", e);
         }
     }
```

Comment:
I guess the original intention is:
```rs
    for inner in rx {
        if let Ok(events) = inner {
            for event in events {
                println!("{:?}", event);
            }
        }
    }
```

Reply:
Oh, I misunderstood the original intention. I fixed. Thanks!